### ISSUE TITLE
missing port in address

### ISSUE THREAD

**Issue by mzupan** [meshbird/meshbird#115]
Getting this on latest from git and also latest release

```
# MESHBIRD_KEY="keyHere" ./meshbird new
2018/09/26 10:51:48 config: config.Config{Key:"keyHere", SeedAddrs:"dc1/10.0.0.1/16,dc2/10.0.0.2", LocalAddr:"10.0.0.1", LocalPrivateAddr:"192.168.0.1", Dc:"dc1", TransportThreads:1, Ip:"10.237.0.1/16", Mtu:9000, Verbose:0}
2018/09/26 10:51:48 skip seed addr 10.0.0.1 because it's local addr
2018/09/26 10:51:48 net resolve tcp addr err: address 192.168.0.1: missing port in address
2018/09/26 10:51:48 net resolve tcp addr err: address 10.0.0.1: missing port in address
2018/09/26 10:51:48 transport thread 0 addr 10.0.0.2 connect err: address 10.0.0.2: missing port in address
2018/09/26 10:51:49 transport thread 0 addr 10.0.0.2 connect err: address 10.0.0.2: missing port in address
2018/09/26 10:51:50 transport thread 0 addr 10.0.0.2 connect err: address 10.0.0.2: missing port in address
2018/09/26 10:51:51 transport thread 0 addr 10.0.0.2 connect err: address 10.0.0.2: missing port in address
2018/09/26 10:51:52 transport thread 0 addr 10.0.0.2 connect err: address 10.0.0.2: missing port in address
2018/09/26 10:51:53 net resolve tcp addr err: address 192.168.0.1: missing port in address
2018/09/26 10:51:53 net resolve tcp addr err: address 10.0.0.1: missing port in address
```

**Comment by Fastidious** [comment]
You will use `meshbird new` to generate a key, and:
```
$ MESHBIRD_KEY="<key>" meshbird join
```
To join a network. See the [website](http://meshbird.com/) for usage.

**Comment by iwater** [comment]
the same error on mac

**Comment by binstreamio** [comment]
same problem on linux

# MESHBIRD_KEY="testtest"  bin/meshbird new    
2018/09/29 15:01:18 config: config.Config{Key:"testtest", SeedAddrs:"dc1/10.0.0.1/16,dc2/10.0.0.2", LocalAddr:"10.0.0.1", LocalPrivateAddr:"192.168.0.1", Dc:"dc1", TransportThreads:1, Ip:"10.237.0.1/16", Mtu:9000, Verbose:0}
2018/09/29 15:01:18 skip seed addr 10.0.0.1 because it's local addr
2018/09/29 15:01:18 transport thread 0 addr 10.0.0.2 connect err: address 10.0.0.2: missing port in address
2018/09/29 15:01:18 net resolve tcp addr err: address 192.168.0.1: missing port in address
2018/09/29 15:01:18 net resolve tcp addr err: address 10.0.0.1: missing port in address
2018/09/29 15:01:19 transport thread 0 addr 10.0.0.2 connect err: address 10.0.0.2: missing port in address

**Comment by matthewgao** [comment]
As per the source code,  I believe "new" is not available parameter for the current v2.0 version, and there is no DHT，so you have to setup the right params by yourself I believe

**Comment by miolini** [comment]
Yes. DHT was disabled by security reasons. Meshbird 3 will have almost similar options.
